src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -1,382 +1,167 @@
+              "actions": {},
+              "outputs": {},
+              "parameters": {},
+              "triggers": {
+                "manual": {
+                  "inputs": {
+                    "schema": {}
+                  },
+                  "kind": "Http",
+                  "type": "Request"
+                }
+              },
+              "contentVersion": "1.0.0.0"
+            }'
+        }
+        Context "Enabling Logic Apps without configuration" {
             It "Enables a specific Logic App"{
                 # Arrange
-                $resourceGroupName = "codit-arcus-scripting"
-                $logicAppName = "arc-dev-we-rcv-http-trigger"
-                Mock Write-Host {}
-                Mock Invoke-WebRequest -MockWith {
-                    return @{
-                        Content = ""
-                        StatusCode = "200"
-                    }
-                }
-                Mock Get-AzCachedAccessToken -MockWith {
-                    return @{
-                        SubscriptionId = "123456"
-                        AccessToken = "accessToken"
-                    }
-                }
-
-                # Act
-                Enable-AzLogicApp -ResourceGroupName $resourceGroupName -LogicAppName $logicAppName
-
-                # Assert
-                Assert-VerifiableMock
-                Assert-MockCalled Get-AzCachedAccessToken -Scope It -Times 1
-                Assert-MockCalled Write-Host -Scope It -Exactly 1 -ParameterFilter { $Object -eq "Successfully enabled arc-dev-we-rcv-http-trigger" }
-            }
-            It "Fails to enable an unknown Azure Logic App" {
-                # Arrange
-                $resourceGroupName = "codit-arcus-scripting"
-                $logicAppName = "arc-dev-we-rcv-unknown-http"
-                $errorContent = "{""error"":{""code"":""ResourceNotFound"",""message"":""Unable to find the resource Microsoft.Logic/workflows/$logicAppName within resourcegroup codit-arcus-scripting.""}}"
-                Mock Write-Warning -MockWith { } -ParameterFilter {$Message -like "Failed to enable $logicAppName"  }
-                Mock Write-Warning -MockWith { } -ParameterFilter {$Message -like "Error: $errorContent"  }
-                Mock Invoke-WebRequest -MockWith {
-                    $status = [System.Net.WebExceptionStatus]::ConnectionClosed
-                    $response = New-Object -type 'System.Net.HttpWebResponse'
-                    $response | Add-Member -MemberType noteProperty -Name 'StatusCode' -Value 404 -force
-                    $exception = New-Object System.Net.WebException $errorContent , $null, $status, $response
-        
-                    Throw $exception
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $logicAppName = Create-AzLogicAppName
+
+                New-AzLogicApp `
```

Comment:
The integration tests now create/remove an unique Logic App instance for each integration test, so it can be perfectly run in parallel with other test runs. Otherwise, we would have unstable enable/disable results if multiple test runs would run against the same set of Logic Apps.

Reply:
Great job in removing them aftewards as well 👍🏻